fluent-bit-firehose.go

```diff
@@ -50,9 +50,9 @@ func getPluginInstance(ctx unsafe.Pointer) *firehose.OutputPlugin {
 }
 
 // The "export" comments have syntactic meaning
-// This is how the compiler knows a function should be callable from the C code
+// This is how the compiler knows a function should be callable from the C code.
 
-//export FLBPluginRegister
+// FLBPluginRegister exporter.
```

Comment:
I believe, the `export` word  in comment has a syntactic meaning. It's a cgo thing.

Reply:
> I believe, the export word in comment has a syntactic meaning. It's a cgo thing.

💯

Reply:
(And this is why we can't run Golint on this file)

Reply:
Thanks for pointing this out. I wasn't aware that these comments had syntactic meaning. I will skip this file for golint checks.